tmt/steps/provision/mrack.py

```diff
@@ -1179,6 +1180,12 @@ def _create(self, tmt_name: str) -> None:
             public_key=self.public_key,
             group=self.beaker_job_group)
 
+        mrack_requirement = self.api._mrack_transformer.create_host_requirement(data)
+        if self.is_dry_run:
+            job = self.api._mrack_provider._req_to_bkr_job(mrack_requirement)
+            self.print(re.sub(r'^\t+', lambda match: '  ' *
```

Comment:
* A comment would be nice, to explain *why*, e.g. "mrack indents XML with tabs, tmt indents with spaces, let's make sure we don't mix these two in tmt output & modify mrack's output to use spaces as well"
* can you try a different line split, a new line after `print(`? Split after `*` separates two operands of multiplication, that's hard to follow and easy to remove `*` some day.

Reply:
@happz pls re-review